<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SFile</class>
 <widget class="QDialog" name="SFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件传输</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择文件/文件夹</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPushButton" name="openBtn">
     <property name="text">
      <string>打开...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="sendBtn">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="msgLabel">
     <property name="text">
      <string>请选择要发送的文件！</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
